Client/Frontend/Strings.swift

```diff
@@ -406,6 +406,7 @@ extension Strings {
     public static let AppMenuAddToReadingListTitleString = NSLocalizedString("Menu.AddToReadingList.Title", tableName: "Menu", value: "Add to Reading List", comment: "Label for the button, displayed in the menu, used to add a page to the reading list.")
     public static let AppMenuShowTabsTitleString = NSLocalizedString("Menu.ShowTabs.Title", tableName: "Menu", value: "Show Tabs", comment: "Label for the button, displayed in the menu, used to open the tabs tray")
     public static let AppMenuSharePageTitleString = NSLocalizedString("Menu.SharePageAction.Title", tableName: "Menu", value: "Share Page With…", comment: "Label for the button, displayed in the menu, used to open the share dialog.")
+    public static let AppMenuCloseTabTitleString = NSLocalizedString("Menu.CloseTabAction.Title", tableName: "Menu", value: "Close Tab", comment: "Label for the button, displayed in the menu, used to close the current tab.")
```

Comment:
This could be replaced by already existing `Strings.CloseTabTitle`.

Reply:
I think it'd be simpler for l10n to reuse the string so we can land this for 10.3

Reply:
:+1: